classes/Song.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import defaults
+import uuid
+import jwt
+import os
+
+
+from db_templates.load_template import load_db_template
+from utilities.db_util import connect_execute_query
+
+
+class Song:
+    def __init__(self, artist, genre, song_name, yt_link):
+        self.artist = artist
+        self.genre = genre
+        self.song_name = song_name
+        self.yt_link = yt_link
+    
+    def check_if_song_exists(self) -> bool:
+        query=f"SELECT yt_link FROM songs WHERE yt_link=\'{self.yt_link}\'"
```

Comment:
Why do we only check for yt link here ? 

If the idea is that yt link is only thing unique about the song - I could have two different links for the same song etc. link with video, and link with official audio

Reply:
I'd rather check for artist + song name (have it all lowercase while checking etc.), we will have duplicates only if users mess up the name, spell it wrong etc